### PR TITLE
[DOCS] SceneState method descriptions added.

### PR DESCRIPTION
**Pull request by willnationsdev**

### CODE REVIEW

File: doc/classes/SceneState.xml [hunk 1]
```diff
@@ -161,6 +181,7 @@
 			<argument index="0" name="idx" type="int">
 			</argument>
 			<description>
+				Returns true if the node at [code]idx[/code] is an [class InstancePlaceholder].
```
Comment (cbscribe) [inline doc/classes/SceneState.xml]: Please put [code] tags around "true"

Comment (willnationsdev) [reply doc/classes/SceneState.xml]: done

File: doc/classes/SceneState.xml [hunk 2]
```diff
@@ -29,6 +33,7 @@
 			<argument index="0" name="idx" type="int">
 			</argument>
 			<description>
+				Returns the flags for the signal at [code]idx[/code]. See [class Object]'s [code]CONNECT_*[/code] flags.
```
Comment (akien-mga) [inline doc/classes/SceneState.xml]: `[Object]`

File: doc/classes/SceneState.xml [hunk 3]
```diff
@@ -91,6 +103,7 @@
 			<argument index="0" name="idx" type="int">
 			</argument>
 			<description>
+				Returns the path to the represented scene file if the node at [code]idx[/code] is an [class InstancePlaceholder].
```
Comment (akien-mga) [inline doc/classes/SceneState.xml]: Remove `class `.

File: doc/classes/SceneState.xml [hunk 4]
```diff
@@ -161,6 +181,7 @@
 			<argument index="0" name="idx" type="int">
 			</argument>
 			<description>
+				Returns [code]true[/code] if the node at [code]idx[/code] is an [class InstancePlaceholder].
```
Comment (akien-mga) [inline doc/classes/SceneState.xml]: Same here.

**Comment by willnationsdev** [comment]
done

**Pull request merged**